release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -71,17 +71,33 @@ For more information, see xref:../installing/cluster-capabilities.adoc#cluster-c
 [id="ocp-4-16-web-console"]
 === Web console
 
+[id="ocp-4-16-patternfly4-deprecated"]
+==== Patternfly 4 is now deprecated with 4.16
+With this release, Patternfly 4 and React Router 5 is depreciated in the web console. All plugins should migrate to Patternfly 5 and React Router 6 as soon as possible.
+
 [id="ocp-4-16-administrator-perspective"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
-//Are there any for 4.16
+
+* A Google Cloud Platform (GCP) token authorization was added to the *Infrastructure features* filter in the OperatorHub.
+* A new quick start is available with information on the impersonate workflow and functionality for users with impersonare access.
 
 [id="ocp-4-16-developer-perspective"]
 ==== Developer Perspective
 
 This release introduces the following updates to the *Developer* perspective of the web console:
-//* are there any for 4.16?
+
+* When searching,
+A new section was added to the *Search* menu to display the recently searched items in the order they were searched.
+* With this release, you can collapse and expand the *Getting started* section.
+
+[id="ocp-4-16-console-telemetry-plugin"]
+===== Console Telemetry plugin
+
+With this release, you can configure your settings to opt in or out to sending Telemetry data to Red Hat. No data will be sent without choosing to opt in.
+
```

Comment:
@jerolimov Do we need to add this in release note since we disabled this feature using flag?

Reply:
@lokanandaprabhu @opayne1 Yes, we should adjust this comment.

The UI is only visible if a customer adds a particular configuration to the cluster. That's not the default. Users will not see a UI for opt in until the cluster admins configure that option.

I think we should mention telemetry in the release note. But we should mention here that we enabled telemetry now also in the front end (= analytics) when the cluster telemetry is enabled.

```suggestion
===== Console Telemetry

With this release, we enabled anonymized frontend telemetry (user analytics) if cluster telemetry is enabled. This is the default for most of the cluster and helps our teams understand how the Console is used.

Cluster administrators can change this per cluster. They can switch to an opt-in, opt-out, or disable this frontend telemetry.
```

Wdyt @lokanandaprabhu @opayne1 @alimobrem @jhadvig ?

/cc @vikram-raj @invincibleJai

Reply:
Updated. Thanks everyone!